styles.css

```diff
@@ -0,0 +1,1916 @@
+    transition: opacity 0.4s;
+
+    --transform: translateY(0);
+    --transition: transform 0.3s 1s;
+  }
+
+  .uber_title {
+    margin-top: -1340px;
+    margin-bottom: 40px;
+    margin-left: -860px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .punto7 {
+    list-style-type: none;
+    margin-left: -545px;
+    margin-top: -37px;
+  }
+
+  .punto8 {
+    list-style-type: none;
+    margin-left: -400px;
+    margin-top: -38px;
+  }
+
+  .backend3 {
+    list-style-type: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #7a869a;
+    margin-left: -530px;
+    margin-top: -35px;
+  }
+
+  .año2 {
+    list-style-type: none;
+    margin-left: -382px;
+    margin-top: -37px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #7a869a;
+  }
+
+  #canopyesc3 {
+    list-style-type: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    margin-top: 35px;
+    margin-left: -590px;
+  }
+
+  .mobile-modal {  /* APAGAR CARD 1 MOBILE EN ESCRITORIO */
+    display: none;
+  }
+
+  .mobile-modal1 { /* APAGAR CARD 2 MOBILE EN ESCRITORIO */
+    display: none;
+  }
+
+  .mobile-modal2 { /* APAGAR CARD 3 MOBILE EN ESCRITORIO */
+    display: none;
+  }
+
+  .mobile-modal3 { /* APAGAR CARD 4 MOBILE EN ESCRITORIO */
+    display: none;
+  }
+
+  .error-message {
+    color: red;
+    margin-top: 5px;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .nohay {
+    display: none; /* no visible en version mobile */
+  }
+
+  .sihay {
+    display: block; /* Visible en mobile */
+  }
+
+  .hamburguer {
+    display: block;
+  }
+
+  .hrupo .bar:hover {
+    background: white;
+  }
+
+  .hamburguer.active .bar:nth-child(2) {
+    opacity: 0;
+    background: white;
+    margin-right: -30px;
+    margin-left: 60px;
+  }
+
+  .hamburguer.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background: white;
+    margin-right: -30px;
+    margin-left: 60px;
+  }
+
+  .hamburguer.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background: white;
+    margin-right: -30px;
+    margin-left: 60px;
+  }
+
+  .nav-menu {
+    position: fixed; /* Background menu */
+    left: -100%;
+    top: 1px;
+    bottom: 0;
+    gap: 0;
+    flex-direction: column;
+    background-color: #6070ff;
+    backdrop-filter: blur(0.5rem);
+    mix-blend-mode: multiply;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+  }
+
+  .nav-item { /* Portafolio */
+    margin: 100px 0;
+    margin-left: 10px;
+  }
+
+  .nav-item2 { /* About */
+    margin-top: -80px;
+    margin-left: -30px;
+  }
+
+  .nav-item3 {
+    margin-top: 25px;
+  }
+
+  .nav-link {  /* Mover todos los nav-link a la izquierda */
+    margin-left: -210px;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
+
+  /* APAGAR VERSION ESCRITORIO */
+
+  .modal {
```

Comment:
- [ ] Great job working on the popup modal for big screens. When I click to get the popup window, the project description, skills, see live and see source buttons don't fit on the page. Try to see how you resize so that everything is visible on the popup as per the figma design.
- Your design
![image](https://github.com/maximoortelli/Portfolio-reactivo/assets/62849872/43cda637-ffcc-466b-b2ab-13cd158aa2d0)
- Figma design
![image](https://github.com/maximoortelli/Portfolio-reactivo/assets/62849872/2a4bfb07-6582-433e-aca5-0eb5147513cb)